pcweb/pages/pricing/calculator.py

```diff
@@ -0,0 +1,291 @@
+        ),
+        class_name="flex flex-row gap-2.5 h-[2.25rem]",
+    )
+
+
+def pricing_widget() -> rx.Component:
+    return rx.box(
+        rx.box(
+            # Team seats
+            calculator(
+                "Members",
+                stepper(
+                    BillingState.estimated_seats,
+                    default_value="1",
+                    min_value=1,
+                    max_value=BillingState.max_seats,
+                    on_click_decrement=BillingState.setvar(
+                        "estimated_seats", (BillingState.estimated_seats - 1)
+                    ),
+                    on_click_increment=BillingState.setvar(
+                        "estimated_seats", (BillingState.estimated_seats + 1)
+                    ),
+                ),
+                f"${BillingState.estimated_seats * BillingState.seat_rate}",
+            ),
+            # GB RAM
+            calculator(
+                "GB RAM",
+                stepper(
+                    BillingState.estimated_ram_gb,
+                    default_value="1",
+                    min_value=1,
+                    max_value=BillingState.max_ram,
+                    on_click_decrement=BillingState.setvar(
+                        "estimated_ram_gb", (BillingState.estimated_ram_gb - 1)
+                    ),
+                    on_click_increment=BillingState.setvar(
+                        "estimated_ram_gb", (BillingState.estimated_ram_gb + 1)
+                    ),
+                ),
+                f"${round(BillingState.estimated_ram_gb * (BillingState.mem_rate * MONTH_MINUTES))}",
+            ),
+            # CPU
+            calculator(
+                "CPU",
+                stepper(
+                    BillingState.estimated_cpu_number,
+                    default_value="0",
+                    min_value=1,
+                    max_value=BillingState.max_cpu,
+                    on_click_decrement=BillingState.setvar(
+                        "estimated_cpu_number", (BillingState.estimated_cpu_number - 1)
+                    ),
+                    on_click_increment=BillingState.setvar(
+                        "estimated_cpu_number", (BillingState.estimated_cpu_number + 1)
+                    ),
+                ),
+                f"${round(BillingState.estimated_cpu_number * (BillingState.cpu_rate * MONTH_MINUTES))}",
+            ),
+            class_name="flex flex-col gap-2",
+        ),
+        # Total 1 month
+        rx.center(
+            rx.flex(
+                rx.badge(
+                        f"Total: ${
+                        round(
+                        BillingState.estimated_seats * BillingState.seat_rate
+                        + BillingState.estimated_ram_gb * (BillingState.mem_rate * MONTH_MINUTES)
+                        + BillingState.estimated_cpu_number * (BillingState.cpu_rate * MONTH_MINUTES)
+                        - 25
```

Comment:
why `-25`? if this is the hobby tier we should have it marked and is the hobby tier really 25 a month?

Reply:
cpu is 20 and 10$ per gig so 5

Reply:
its just the usage from the free tier